HTML/index.html

```diff
@@ -33,12 +33,120 @@
       </div>
 
       <div class="row">
-        <div class="col-sm-8">
-          <img src="math.png" alt="damn damn" width=100% height=40%/>
+	  
+	  
+	          <div class="col-sm-4">
+		<div class="Current">
+          <table style="width:350px; float: center; text-align: center;">
```

Comment:
Small changes needed:

Let's keep html classes lowercased: `Current` -> `current`.

Another thing is avoiding inline styles. the `table` should be given a class name and the styles then moved to the css file.